package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
```

Comment:
Perfect! It fixes the thing. In fact this way allows to remove `npm-run-all` dependency. Works like a charm for me on linux. Cannot test it on windows at the moment, hope it is ok there as well. If not, then I'll put `npm-run-all` back. Gonna check it later on.